src/plugins/upf/test/test_upf.py

```diff
@@ -0,0 +1,300 @@
+from datetime import datetime, timedelta
+import uuid
+import framework
+from scapy.contrib.pfcp import *
+from scapy.layers.l2 import Ether
+from scapy.layers.inet import IP, UDP
+
+def seid():
+    return uuid.uuid4().int & (1<<64)-1
+
+DROP_IP = "192.168.100.99"
+REDIR_IP = "192.168.100.100"
+REDIR_TARGET_IP = "192.168.42.42"
```

Comment:
maybe pick IPs from the TEST-NET range to avoid any issues on $setups out there:

https://en.wikipedia.org/wiki/Reserved_IP_addresses#IPv4

Reply:
Ok, will fix that, although in this case it doesn't really matter as the tests run on "purely imaginary" network, that is, pcap files that are generated by the test are injected on some VPP interfaces and then pcaps are captured on the others, without even touching any real network interface.

Reply:
agreed, not a crucial change or the like … it is just from the right range